### DISCUSSION TITLE
流水线一级页面列表模式增加自定义字段信息展示

### DISCUSSION THREAD

**Discussion by curlychen** [Tencent/bk-ci#4919] [Ideas]
![image](https://user-images.githubusercontent.com/24561462/128980743-935b03bb-282f-4077-a5e7-ec940c505225.png)
curlychen(陈浩) 8-11 11:53:08
这里是可以自定义吗

curlychen(陈浩) 8-11 11:53:22
我想加一个最近一次的执行人

curlychen(陈浩) 8-11 11:54:06
我没找到这个

**Comment by zanyzhao** [comment]
#3914 已经有类似issue了，请关注
